src/components/Contact/styles.scss

```diff
@@ -0,0 +1,103 @@
+@import 'theme/colors.scss';
+@import 'theme/spacings.scss';
+@import 'theme/shadows.scss';
+
+.contact {
+  display: flex;
+  flex-flow: column nowrap;
+
+  width: 100%;
+  width: 100vw;
+
+  padding: $spc-l 0 $spc-xxl 0;
+
+  background: $secondary;
+  align-items: center;
+}
+
+.title {
+  color: $accent;
+
+  text-align: center;
+  text-transform: uppercase;
+  text-shadow: $heading-shadow;
+}
+
+.form {
+  display: flex;
+  flex-flow: column nowrap;
+
+  width: 80%;
+  width: 80vw;
+
+  margin-top: $spc-m;
+}
+
+.input {
+  height: 3.125rem; //50px
+
+  padding: $spc-s;
+  margin-bottom: $spc-s;
+
+  background: $form-color;
+
+  font-size: 1rem;
```

Comment:
Shouldn't this be default? Since root element has `1rem` and all children will refer to that.